src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -97,11 +101,15 @@ public PublicClientApplicationBuilder WithIosKeychainSecurityGroup(string keycha
         }
 
         /// <summary>
-        ///
+        /// On Android and iOS, brokers enabled Single-Sign-On, device identification,
+        /// and application identification verificatoin. To enable one of these features,
+        /// you need to set the WithBroker() parameters to true. See https://aka.ms/msal-net-brokers 
```

Comment:
This wiki page needs to be created.
We also need to update the Mobile scenario landing page: See https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-mobile-overview and the following pages.

Reply:
yes....i have to created the aka.ms links yet, as I wasn't sure where we want which content.

---
In reply to: [300966352](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1251#discussion_r300966352) [](ancestors = 300966352)